qualang_tools/plot/__init__.py

```diff
@@ -1,3 +1,16 @@
 from qualang_tools.plot.plot import interrupt_on_close
+from qualang_tools.plot.Fitting.fitting import *
+from qualang_tools.plot.Fitting.fitting_functions import *

```

Comment:
Unless we want the user to have access directly to the `fitting_functions`, this should be removed

Reply:
Do we want? what are the considerations?

Reply:
I think it's a bit redundant to be able to access these functions from two different places... But you've made it that way, so I was mainly asking you

Reply:
Do you think it is better to put everything on the same script?

Reply:
yes